src/style.css

```diff
@@ -0,0 +1,31 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  /* display: flex; */
+  background-image: url("./images/star.svg");
+  background-position: center;
+  background-repeat: no-repeat;
+  /* background-size: contain; */
+  /* background-size: 13px 13px; */
+  background-size: auto;
+  color: #000;
```

Comment:
why do you need this?

Reply:
/* background-size: 13px 13px; */
  background-size: auto;

it was for making stars smaller than the container 
but it works now without that properties 

color: #000;
from Figma - removing too